src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1188,9 +1254,29 @@ void MarkEntireAssembly (AssemblyDefinition assembly)
 				MarkEntireType (type, includeBaseTypes: false, new DependencyInfo (DependencyKind.TypeInAssembly, assembly), null);
 		}
 
+		void ProcessDescriptorXml (AssemblyDefinition assembly)
+		{
+			if (Annotations.ProcessedDescriptorXml (assembly))
+				return;
+
+			new EmbeddedXmlStep (assembly).ProcessDescriptors (_context, this);
+		}
+
+		void ProcessAssemblySteps (AssemblyDefinition assembly)
```

Comment:
```suggestion
		void ProcessPerAssemblySteps (AssemblyDefinition assembly)
```
To match the change above. Or ProcessStepsForAssembly?